content/toc-doc.json

```diff
@@ -1,275 +1,399 @@
+            "title": "next-tinacms-markdown"
           },
           {
-            "id": "/docs/plugins/fields/date",
-            "slug": "/docs/plugins/fields/date",
-            "title": "Date & Time"
+            "id": "/packages/gatsby-plugin-tinacms",
+            "slug": "/packages/gatsby-plugin-tinacms",
+            "title": "gatsby-plugin-tinacms"
           },
           {
-            "id": "/docs/plugins/fields/markdown",
-            "slug": "/docs/plugins/fields/markdown",
-            "title": "Markdown"
+            "id": "/packages/gatsby-tinacms-git",
+            "slug": "/packages/gatsby-tinacms-git",
+            "title": "gatsby-tinacms-git"
           },
           {
-            "id": "/docs/plugins/fields/html",
-            "slug": "/docs/plugins/fields/html",
-            "title": "HTML"
+            "id": "/packages/gatsby-tinacms-json",
+            "slug": "/packages/gatsby-tinacms-json",
+            "title": "gatsby-tinacms-json"
           },
           {
-            "id": "/docs/plugins/fields/custom-fields",
-            "slug": "/docs/plugins/fields/custom-fields",
-            "title": "Custom Fields"
+            "id": "/packages/gatsby-tinacms-remark",
+            "slug": "/packages/gatsby-tinacms-remark",
+            "title": "gatsby-tinacms-remark"
           }
         ]
-      },
-      {
-        "title": "Content Creators",
-        "id": "content-creator",
-        "slug": "/docs/plugins/content-creators"
-      },
-      {
-        "title": "Screens",
-        "id": "screens",
-        "slug": "/docs/plugins/screens"
-      },
-      {
-        "title": "Toolbar Widgets",
-        "id": "toolbar:widget",
-        "slug": "/docs/plugins/toolbar-widgets"
-      }
-    ]
-  },
-  {
-    "id": "events",
-    "title": "Events",
-    "items": [
-      {
-        "id": "/docs/events",
-        "slug": "/docs/events",
-        "title": "About Events"
       }
     ]
   },
   {
-    "title": "Media",
-    "id": "media",
+    "category": "Tina Cloud",
+    "slug": "/docs/tina-cloud",
+    "description": "Powerful Git-based content management for teams",
     "items": [
       {
-        "id": "/docs/media",
-        "slug": "/docs/media",
-        "title": "About Media"
-      }
-    ]
-  },
-  {
-    "id": "apis",
-    "title": "External APIs",
-    "items": [
+        "id": "/docs/tina-cloud",
+        "slug": "/docs/tina-cloud",
+        "title": "Overview"
+      },
       {
-        "id": "/docs/api",
-        "slug": "/docs/apis",
-        "title": "About APIs"
-      }
-    ]
-  },
-  {
-    "title": "Integrations",
-    "id": "nextjs",
-    "items": [
+        "id": "/docs/tina-cloud/dashboard",
+        "slug": "/docs/tina-cloud/dashboard",
+        "title": "Dashboard"
+      },
       {
-        "id": "/docs/integrations/nextjs",
-        "slug": "/docs/integrations/nextjs",
-        "title": "Next.js"
+        "id": "/docs/tina-cloud/client",
```

Comment:
When reading over it might make more sense to have the cli be first in the list since one can not do anything with the client unless they have a schema?